minimalkv/net/boto3store.py

```diff
@@ -111,25 +112,53 @@ def __init__(
             if bucket not in s3_resource.buckets.all():
                 raise ValueError("invalid s3 bucket name")
         self.bucket = bucket
-        self.prefix = prefix.strip().lstrip("/")
+
+        if prefix is not None:
+            import warnings
+
+            warnings.warn(
+                "The prefix argument is deprecated and will be removed in a future version. "
+                "Use object_prefix instead.",
+                DeprecationWarning,
+            )
+            object_prefix = object_prefix or prefix
+        self.object_prefix = object_prefix.strip().lstrip("/")
```

Comment:
shouldn't this be private? otherwise we might have to break the API again at some point.